src/JsonSchema/Validator.php

```diff
@@ -26,6 +26,7 @@ class Validator extends BaseConstraint
     const SCHEMA_MEDIA_TYPE = 'application/schema+json';
 
     const ERROR_NONE                    = 0x00000000;
+    const ERROR_ALL                     = ~self::ERROR_NONE;
```

Comment:
Forgot that *&@^$ no-expressions-in-constant-definitions thing though. Have changed it to 0xFFFFFFFF, which achieves exactly the same thing.